order-history-text-search-service/src/main/java/io/eventuate/examples/tram/ordersandcustomers/orderhistorytextsearch/OrderHistoryTextSearchServiceMain.java

```diff
@@ -0,0 +1,20 @@
+package io.eventuate.examples.tram.ordersandcustomers.orderhistorytextsearch;
+
+import io.eventuate.examples.tram.ordersandcustomers.commonswagger.CommonSwaggerConfiguration;
+import io.eventuate.examples.tram.ordersandcustomers.orderhistorytextsearch.backend.OrderHistoryTestSearchConfiguration;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.actuate.autoconfigure.elasticsearch.ElasticSearchRestHealthContributorAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.context.annotation.Import;
+
+@SpringBootApplication(exclude = {
+        ElasticSearchRestHealthContributorAutoConfiguration.class,
+        DataSourceAutoConfiguration.class
```

Comment:
Why are JDBC jars are the classpath and triggering `DataSourceAutoConfiguration`?

Reply:
common module has org.springframework.boot:spring-boot-starter-data-jpa it is triggering autoconfiguration. There are common classes like Money and OrderDetails. They use jpa annotations. Also they are used as event fields.